bigquery_etl/metadata/parse_metadata.py

```diff
@@ -198,6 +198,7 @@ class Metadata:
     deletion_date: Optional[date] = attr.ib(None)
     monitoring: Optional[MonitoringMetadata] = attr.ib(None)
     require_column_descriptions: bool = attr.ib(False)
+    level: bool = attr.ib(False)
```

Comment:
Is False a valid value for level? Or should it be None to indicate a lack of assigned value?

Reply:
It should be None, corrected.